src/Core/Utility/Form.php

```diff
@@ -42,7 +42,7 @@ class Form
      * @param mixed|null $value Property value.
      * @return array
      */
-    public static function control(array $schema, string $type, $value) : array
+    public static function control(array $schema = [], string $type, $value) : array
```

Comment:
Not a good practice: default arguments should be on the right side of any non-default arguments.
See http://php.net/manual/en/functions.arguments.php `example #5`